lib/classes/Variables.js

```diff
@@ -20,7 +20,7 @@ class Variables {
     this.selfRefSyntax = RegExp(/^self:/g);
     this.cfRefSyntax = RegExp(/^cf:/g);
     this.s3RefSyntax = RegExp(/^s3:(.+?)\/(.+)$/);
-    this.stringRefSynax = RegExp(/'.+'/g);
+    this.stringRefSynax = RegExp(/('|").+('|")/g);
```

Comment:
A simple empty string, `''`, is often a valid default. Maybe change the regex match from `.+` to `.*`?